ceph-medic.spec.in

```diff
@@ -56,7 +47,6 @@ An admin tool to determine common issues on Ceph storage clusters.
 
 %install
 python setup.py install --prefix=%{_prefix} --root=%{buildroot}
-install -m 0755 -D scripts/ceph-medic $RPM_BUILD_ROOT/usr/bin
```

Comment:
There is no ``scripts/ceph-medic`` directory to install here. This was cruft from ceph-deploy I believe.